### PR TITLE
Adding jfa-go and docs

### LINKED ISSUE #1186: Add jfa-go
# Feature request

https://github.com/hrfee/jfa-go

Makes user management in Jellyfin easier. Opening issue to keep track of it

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1186

### CODE REVIEW

File: compose/.apps/jfago/jfago.labels.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+services:
+  jfago:
+    labels:
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.description: "User management app for Jellyfin"
+      com.dockstarter.appinfo.nicename: "jfa-go"
```
Comment (nemchik) [inline compose/.apps/jfago/jfago.labels.yml:6]: `nicename` has to be `jfago` or some variant with caps like `JFAgo` or something. has to match the var names.

File: compose/.apps/jfago/jfago.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+services:
+  jfago:
+    container_name: jfago
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
```
Comment (nemchik) [inline compose/.apps/jfago/jfago.yml:6]: doesn't look like this image uses PUID/PGID

**Review by nemchik — CHANGES_REQUESTED** [review]

File: compose/.apps/jfago/jfago.yml (hunk 2 above)
Comment (angrycuban13) [reply compose/.apps/jfago/jfago.yml]: 25c8e68

File: compose/.apps/jfago/jfago.labels.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/jfago/jfago.labels.yml]: afb2186

**Comment by nemchik** [comment]
Missing nav link here too 😄

**Comment by angrycuban13** [comment]
Oh no! Lol, fixed it in all 3.

**Review by nemchik — APPROVED** [review]

**Pull request merged**